CRM/Event/DAO/Event.php

```diff
@@ -6,7 +6,7 @@
  *
  * Generated from xml/schema/CRM/Event/Event.xml
  * DO NOT EDIT.  Generated by CRM_Core_CodeGen
- * (GenCodeChecksum:164ee1a507b9244536114d1569ccff1e)
+ * (GenCodeChecksum:82ba48cbb804cf6f4b26fa50f07d44db)
```

Comment:
@seamuslee001 Why is this in here?

Reply:
Because it is there I'll spin that off separately